solutions/src/main/java/ru/sberbank/school/task03/FxClientControllerImpl.java

```diff
@@ -0,0 +1,79 @@
+package ru.sberbank.school.task03;
+
+import ru.sberbank.school.task02.*;
+import ru.sberbank.school.task02.exception.FxConversionException;
+import ru.sberbank.school.task02.util.*;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class FxClientControllerImpl implements FxClientController {
+
+    FxConversionService fxConversionService;
+
+    public FxClientControllerImpl(FxConversionService fxConversionService) {
+        this.fxConversionService = fxConversionService;
+    }
+
+    @Override
+    public List<FxResponse> fetchResult(List<FxRequest> requests) {
+
+        List<FxResponse> responses = new ArrayList<>();
+
+        for (FxRequest r : requests) {
+            responses.add(fetchResult(r));
+        }
+
+        return responses;
+    }
+
+    @Override
+    public FxResponse fetchResult(FxRequest requests) throws FxConversionException {
+
+        if (requests == null) throw new FxConversionException("Некорректный запрос");
+
+        if (requests.getSymbol() == null) throw new FxConversionException("Некорректный запрос");
+        String symbolAsString = requests.getSymbol();
+        if (requests.getDirection() == null) throw new FxConversionException("Некорректный запрос");
+        String directionAsString = requests.getDirection();
+        if (requests.getAmount() == null) throw new FxConversionException("Некорректный запрос");
```

Comment:
&#39;if&#39; construct must use &#39;{}&#39;s.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1295739666>*